js/site-nav.js

```diff
@@ -4,35 +4,44 @@
 
 var $ = require('jquery');
 
+/** SiteNav module
```

Comment:
Extremely minor, but jsdoc-style comments do spacing just a little differently--we'd want to dedent lines 8-12 by one space. If you're using sublime, check out https://github.com/spadgos/sublime-jsdocs or something similar.